wisp-launchdarkly/src/main/kotlin/wisp/launchdarkly/LaunchDarklyClientMetrics.kt

```diff
@@ -0,0 +1,30 @@
+package wisp.launchdarkly
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.DistributionSummary
+import io.micrometer.core.instrument.MeterRegistry
+
+class LaunchDarklyClientMetrics(meterRegistry: MeterRegistry) {
+
+  internal fun initializationDuration(
+    meterRegistry: MeterRegistry
+  ): DistributionSummary =
+    DistributionSummary.builder(HISTO_DURATION_NAME)
+      .description("count and duration in ms of ld client initialization")
+      .publishPercentileHistogram()
+      .register(meterRegistry)
+
+  internal val successCount: Counter = Counter.builder(SUCCESS_COUNTER_NAME)
+    .description("count of successful ld client initializations")
+    .register(meterRegistry)
+
+  internal val failedCount: Counter = Counter.builder(FAILED_COUNTER_NAME)
+    .description("count of ld client initialization failures")
+    .register(meterRegistry)
+
+  companion object {
+    const val HISTO_DURATION_NAME = "histo_ld_initialization_duration_ms"
+    const val SUCCESS_COUNTER_NAME = "ld_initialization_success"
+    const val FAILED_COUNTER_NAME = "ld_initialization_failed"
```

Comment:
Should these have "_total" for compatibility with prometheus?

Reply:
added.